graphql/resolvers/addComment.ts

```diff
@@ -11,6 +11,129 @@ import {
   PRIMARY_COLOR_HEX,
   PROFILE_URL
 } from '../../constants'
+import { Challenge, Comment, Lesson, Submission, User } from '@prisma/client'
+
+type CommentWithAuthorAndSubmission = Comment & {
+  author: {
+    id: number
+    username: string
+    discordId: string | null
+    name: string
+  }
+  submission: Submission & {
+    user: { id: number; username: string; discordId: string | null }
+    lesson: Lesson
+    challenge: Challenge
+  }
+}
+
+type SendNotificationsParams = {
+  line: InputMaybe<number> | undefined
+  fileName: InputMaybe<string> | undefined
+  submissionId: number
+  comment: CommentWithAuthorAndSubmission
+}
+
+const sendNotifications = async ({
+  line,
+  submissionId,
+  fileName,
+  comment
+}: SendNotificationsParams) => {
+  const restOfComments = await prisma.comment.findMany({
```

Comment:
Do not request all the comment data, you only need the author id, author username and author discordIt, no other field is needed on the author on comment, only request the minimum amount of needed data.